.github/PULL_REQUEST_TEMPLATE/standard-change.md

```diff
@@ -4,7 +4,7 @@
 
 ## Related links
 
-<!-- Write the links related to this PR. -->
+<!-- Write the links related to this PR. Private links should be clearly marked as private, for example, 'TIER IV INTERNAL LINK'. -->
```

Comment:
Sorry, I should have written a URL.

```suggestion
<!-- Write the links related to this PR. Private links should be clearly marked as private, for example, '[TIER IV INTERNAL LINK] https://example.com'. -->
```

Reply:
By the way, it might be better to remove TIER IV.

Reply:
@shmpwk I'm sorry, could you check it again? :pray:

Reply:
How about 

```
'[FOO COMPANY INTERNAL LINK](https://example.com)'
```
instead of 
```
'[FOO COMPANY INTERNAL LINK] https://example.com'
```
??

Reply:
Yeah, it's fine. I'll leave it to you.

Reply:
Reflected by
https://github.com/autowarefoundation/autoware/pull/2877/commits/19f09dbbafbb854cee8f4d34263004cb6ba5c163